l1-contracts/contracts/upgrades/BaseZkSyncUpgradeGenesis.sol

```diff
@@ -14,6 +14,7 @@ abstract contract BaseZkSyncUpgradeGenesis is BaseZkSyncUpgrade {
     function _setNewProtocolVersion(uint256 _newProtocolVersion) internal override {
         uint256 previousProtocolVersion = s.protocolVersion;
         require(
+            // !! ONLY CHANGE IS IN THIS LINE !!
```

Comment:
People did not notice that the following line changed, so I put an extra notice there

Reply:
Sorry, this looks a bit dirty and probably wont be understandable for those outside of context.

Let's do the following:

- Delete `function upgrade(` from this file. Since this function is the same as in the parent contract, the overridden `_setNewProtocolVersion` will be used in it. This way the smaller diff will be more apparent.
- You can put one capslock word `IMPORTANT` into the comment right below since it already describes the difference